src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -183,6 +183,9 @@ function ($m) {
 
         restore_error_handler();
 
+        // Remove NULL characters to prevent DOMDocument::loadHTML() from stopping the document parsing
+        $content = str_replace("\x00", '', $content);
```

Comment:
We don't ship workarounds for PHP bugs unless they have an impact for a large number of our users. That's not the case here, so, I think we can now close this one.